src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -788,27 +906,35 @@ private Traverse GetTraverse(string direction, IVariableInformation node)
             return go;
         }
 
-        private class Node
+        /// <summary>
+        /// Traverse tree based on specified startIndex.
+        /// Then add wrappers for Natvis tree visualizations.
+        /// </summary>
+        /// <param name="root">Root of the tree</param>
+        /// <param name="goLeft">Left child of root</param>
+        /// <param name="goRight">Right child of root</param>
+        /// <param name="getValue">Root value</param>
+        /// <param name="content">List of IVariableInformation children</param>
```

Comment:
`content` is more like the list of variables to be displayed given the current variable.

Reply:
Will fix. Thanks.